src/FondOfSpryker/Glue/CompanyUserCompanyTypeRolesRestApi/Processor/Validator/CompanyUserCompanyTypeRolesSearchValidator.php

```diff
@@ -85,8 +87,11 @@ public function validate(CompanyUserTransfer $companyUserTransfer): bool
             return false;
         }
 
-        if ($this->hasCompanyUserValidCompanyTypeRoles($companyRoleCollectionTransfer, $companyTypeTransfer) === false) {
-            return false;
+        if ($this->hasCompanyUserValidCompanyTypeRoles(
```

Comment:
return $this->hasCompanyUserValidCompanyTypeRoles($companyRoleCollectionTransfer, $companyTypeResponseTransfer->getCompanyTypeTransfer());

Reply:
done